invenio_users_resources/resources/users/resource.py

```diff
@@ -78,3 +82,39 @@ def avatar(self):
             last_modified=avatar.last_modified,
             max_age=avatar.max_age,
         )
+
+    @request_view_args
```

Comment:
We should add an error handler for lock errors.

This is useful for the caller to know why the moderation failed.

Since locks are not merged yet, I opened the PR without this handler. It can be added later on.